setup.py

```diff
@@ -14,7 +14,7 @@
 
 
 setup(name="smatch",
-      version="1.0",
+      version=os.environ.get("TRAVIS_TAG") or "1.0",
```

Comment:
This means the version will be `1.0` except when Travis-CI is running it (or someone configures the TRAVIS_TAG environment variable themself). Instead, [setuptools_scm](https://github.com/pypa/setuptools_scm) seems like a more general solution. This would help people who install from source to have an appropriate version, too.